source/core_json.c

```diff
@@ -73,7 +73,7 @@ static void skipSpace( const char * buf,
                        size_t * start,
                        size_t max )
 {
-    size_t i;
+    size_t i = 0U;
```

Comment:
I'm fairly certain you don't need to mark 0 as 0U?
I believe it's unsigned by default in C

Reply:
Misra 2012 requires U. signed is default.

Reply:
Literal 0 is an integer constant of type "signed int" ([ISO:C90 6.1.3.2 - Semantics](https://www.yodaiken.com/wp-content/uploads/2021/05/ansi-iso-9899-1990-1.pdf)).